### ISSUE TITLE
option expireToken on connect has no effect

### ISSUE THREAD

**Issue by gloced** [GetStream/stream-js#238]
I realised that the available option _expireTokens_ has no effect when creating a client token.
The resulting JWT has no expire field, hence tokens are valid forever.

init of stream:
`const client = stream.connect(streamKey, streamSecret, streamAPPID, {location: 'dublin', expireTokens: true});`

geting token:
`client.createUserToken('userid');`

**Comment by JelteF** [comment]
Thanks for the bugreport. I just fixed this in v4.2.1

One thing to keep in mind is that you probably also want to add the [`exp` claim](https://tools.ietf.org/html/rfc7519#section-4.1.4) when creating the token, e.g:
```js
client.createUserToken('userid', {exp: Math.floor(Date.now()/1000) + 3600});
```

**Comment by gloced** [comment]
works great, thanks for the hint with the "exp" option on creating the token
